package.json

```diff
@@ -50,10 +51,14 @@
     "eslint": "8.13.0",
     "eslint-config-airbnb-base": "15.0.0",
     "eslint-plugin-import": "2.26.0",
+    "http": "^0.0.1-security",
+    "https": "^1.0.0",
```

Comment:
Is this required to make TypeScript work? These two dependencies (`http` and `https` seem to be placeholders) and I wonder if we could introduce these externals differently (for example, could we tell tsconfig that we're working in a NodeJS environment?)